### DISCUSSION TITLE
MDX_Vault\MDX_Vault DX7II\ROMs\VRC-1005 Studio

### DISCUSSION THREAD

**Discussion by Piattica** [BobanSpasic/MDX_Vault#14] [General]
some performances have conversion errors

**Comment by BobanSpasic** [comment]
Please, use https://github.com/BobanSpasic/MDX_Vault/issues to report bugs/errors/mistakes.
And please, give some more details, or at least which voice(s) and what kind of errors.

Thank you for your hard work on finding errors, but please give more details. I do not have the time, neither the possibilities, to search for undefined errors. I can only do some tests on weekends.

**Reply by Piattica** [reply]
Thank you for your suggestion, I will do it
